app/src/main/assets/css/wysiwyg.css

```diff
@@ -201,4 +201,16 @@
       transform: scale(1.1);
       content: url("file:///android_asset/web/images/cube-scan.svg");
       float: right;
+    }
+
+    blockquote {
+        margin: 0;
+        padding-left: 1em;
+        border-left: 2px solid #ccc;
+        color: #666;
```

Comment:
hardcoded color will not work well in dark mode

Reply:
Added `base-style.css` and `base-style-night.css` with colors for light and dark mode respectively.